x-pack/plugins/uptime/public/components/overview/monitor_list/monitor_list.tsx

```diff
@@ -72,104 +92,113 @@ export const MonitorListComponent: ({
-      width: '30%',
-      render: (url: string) => (
-        <EuiLink href={url} target="_blank" color="text" external>
-          {url}
-        </EuiLink>
-      ),
-    },
-    {
-      align: 'left' as const,
-      field: 'state.monitor.name',
-      name: TAGS_LABEL,
-      width: '12%',
-      render: (_name: string, summary: MonitorSummary) => <MonitorTags summary={summary} />,
-    },
-    {
-      align: 'left' as const,
-      field: 'state.tls.server.x509',
-      name: labels.TLS_COLUMN_LABEL,
-      render: (x509: X509Expiry) => <CertStatusColumn expiry={x509} />,
-    },
-    {
-      align: 'center' as const,
-      field: 'histogram.points',
-      name: labels.HISTORY_COLUMN_LABEL,
-      mobileOptions: {
-        show: false,
+      {
+        align: 'left' as const,
+        field: 'state.monitor.name',
+        name: TAGS_LABEL,
+        width: '12%',
+        render: (_name: string, summary: MonitorSummary) => <MonitorTags summary={summary} />,
       },
-      render: (histogramSeries: HistogramPoint[] | null, summary: MonitorSummary) => (
-        <MonitorBarSeries histogramSeries={histogramSeries} minInterval={summary.minInterval!} />
-      ),
-    },
-    {
-      align: 'center' as const,
-      field: '',
-      name: STATUS_ALERT_COLUMN,
-      width: '100px',
-      render: (item: MonitorSummary) => (
-        <EnableMonitorAlert
-          monitorId={item.monitor_id}
-          selectedMonitor={item.state.summaryPings[0]}
-        />
-      ),
-    },
-    {
-      align: 'right' as const,
-      field: 'monitor_id',
-      name: '',
-      sortable: true,
-      isExpander: true,
-      width: '40px',
-      render: (id: string) => {
-        return (
-          <EuiButtonIcon
-            aria-label={labels.getExpandDrawerLabel(id)}
-            data-test-subj={`xpack.uptime.monitorList.${id}.expandMonitorDetail`}
-            iconType={drawerIds.includes(id) ? 'arrowUp' : 'arrowDown'}
-            onClick={() => {
-              if (drawerIds.includes(id)) {
-                updateDrawerIds(drawerIds.filter((p) => p !== id));
-              } else {
-                updateDrawerIds([...drawerIds, id]);
-              }
-            }}
+      {
+        align: 'left' as const,
+        field: 'state.tls.server.x509',
+        name: labels.TLS_COLUMN_LABEL,
+        render: (x509: X509Expiry) => <CertStatusColumn expiry={x509} />,
+      },
+    ],
+    ...(!hideExtraColumns
+      ? [
+          {
+            align: 'left' as const,
+            field: 'histogram.points',
+            name: labels.HISTORY_COLUMN_LABEL,
+            mobileOptions: {
+              show: false,
+            },
+            render: (histogramSeries: HistogramPoint[] | null, summary: MonitorSummary) => (
+              <MonitorBarSeries
+                histogramSeries={histogramSeries}
+                minInterval={summary.minInterval!}
+              />
+            ),
+          },
+        ]
+      : []),
+    ...[
+      {
```

Comment:
I wish there were a cleaner way to preserve the order of these items, but this works fine.